### PR TITLE
Fixed 'find next focusable' mechanism (closes #139)

### LINKED ISSUE #139: The 'Find next focusable' mechanism (Tab key) doesn't skip elements with 'display=none' in IE and Firefox
T297258

### PR DESCRIPTION
**Pull request by AndreyBelym**
\cc @inikulin @AlexanderMoskovkin

### CODE REVIEW

File: src/client/core/utils/dom.js [hunk 1]
```diff
@@ -221,17 +221,25 @@ function sortBy (property) {
 
 function getAllFocusableElements ($iframe) {
     var $allFocusable = $();
+    var $invisibleElements = $();
 
     if ($iframe) {
         //NOTE: We can get elements of the same domain iframe only
         try {
             $allFocusable = $iframe.contents(0).find(getFocusableSelector());
+            $invisibleElements = $iframe.contents(0).find('*').filter(function(){
```
Comment (inikulin) [inline src/client/core/utils/dom.js]: Here and below: use arrows

**Comment by inikulin** [comment]
\r-

**Comment by inikulin** [comment]
ping @AlexanderMoskovkin

**Comment by AlexanderMoskovkin** [comment]
I am waiting for changing functions to arrows or it's not necessary here?

**Comment by inikulin** [comment]
@AlexanderMoskovkin Argh, yes, right

**Comment by inikulin** [comment]
@AlexanderMoskovkin I just rememberd, we need `this` in jquery filter, so we can't use arrow here. So, we can merge

**Comment by AlexanderMoskovkin** [comment]
what about 
```js
.filter((index, el) => el.style.display === 'none');
```
? It's better then `this` using imho

**Comment by inikulin** [comment]
@AlexanderMoskovkin I like it

**Comment by AndreyBelym** [comment]
FPR

File: src/client/core/utils/dom.js [hunk 2]
```diff
@@ -222,45 +216,36 @@ function sortBy (property) {
...
-        });
+        .filter((index, el) => $(el).attr("tabIndex") !== -1);
 
-    //NOTE: <option> element visible/ hidden in all browser differently
-    // http://api.jquery.com/hidden-selector/
-    if (browserUtils.isWebKit || browserUtils.isOpera) {
-        var $hidden = $allFocusable.filter(function () {
-            return ($(this).is(":hidden") && !($(this).is("option")));
-        });
+    $allFocusable = $allFocusable.filter((index, el) => el.style.display !== 'none' && !$invisibleElements.has(el).length);
```
Comment (AlexanderMoskovkin) [inline src/client/core/utils/dom.js]: The line seems to long. I'd prefer
```js
$allFocusable = $allFocusable.filter((index, el) => {
    return el.style.display !== 'none' && !$invisibleElements.has(el).length);
};
```
@inikulin are you agree or it's ok?

Comment (inikulin) [inline src/client/core/utils/dom.js]: It feets diff window, so it's fine as is.

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by inikulin** [comment]
lgtm

**Pull request merged**